src/coreclr/nativeaot/Runtime/unix/UnixNativeCodeManager.cpp

```diff
@@ -1248,7 +1248,7 @@ bool UnixNativeCodeManager::GetReturnAddressHijackInfo(MethodInfo *    pMethodIn
     }
 
     *ppvRetAddrLocation = (PTR_PTR_VOID)pRegisterSet->pLR;
-#elif
+#else
```

Comment:
It would be nice to avoid the code duplication here instead. I have commented on it in #111103.